hardware/products/ucm/g400.md

```diff
@@ -1,5 +1,250 @@
 # G400
 ![G400S](images/g400s.jpg) ![G400D](images/g400d.jpg)
+# Overview
+---
+The G400-D System on Module (SoM) is the precursor to the Universal Compute Modules (UCM). It pioneered the SO-DIMM System on Module (SoM) originally running .Net Micro Framework exclusively. It now can operate various other platforms, including TinyCLR.
```

Comment:
I believe it should be ".NET Micro Framework", verify. Also "TinyCLR OS", not TinyCLR.

Reply:
Verified and changed.